python/usb_console/simulation.py

```diff
@@ -87,28 +86,34 @@ def run(self):
         while step < num_steps and self.running:
             self.sim_time = self.main_state['follower']['dynamics']['time']
 
-            # Get sensor readings
+            # Step 1. Get sensor readings from simulation
             self.sensor_readings_follower = self.eng.sensor_reading(self.main_state['follower'],self.main_state['leader'], nargout=1)
             self.sensor_readings_leader = self.eng.sensor_reading(self.main_state['leader'],self.main_state['follower'], nargout=1)
-            # update dynamics
+            
+            # Step 2. Update dynamics
             self.main_state = self.eng.main_state_update(self.main_state, nargout=1)
```

Comment:
@nhz2 How do I get this to run in the background?

Reply:
@tanishqaggarwal It's a futures type thing. See the documentation here:
https://www.mathworks.com/help/matlab/matlab_external/call-matlab-functions-asynchronously-from-python.html

Reply:
Thanks!